api_handlers/zotero.php

```diff
@@ -10,7 +10,7 @@ function query_url_api($ids, $templates) {
     if ($failures >= 5 ) {
       // Repeat failures for errors on our side: probably a chronic timeout.
       report_action("Giving up on generic URLs for this page.")
-      break;
+      return;
```

Comment:
No, this is not what I intended. In my example, the rest of the operation continued correctly. Please don't skip the entire thing.